src/main/java/seedu/address/ui/CartCard.java

```diff
@@ -11,7 +11,7 @@
  */
 public class CartCard extends UiPart<Region> {
 
-    private static final String FXML = "CartListCard.fxml";
+    private static final String FXML = "CartCard.fxml";
```

Comment:
This is almost the same as the `InventoryCard` right? Would it make sense to combine them into an `IngredientCard`, since it's for displaying an ingredient? Then the `CartPanel` and `InventoryPanel` can handle `IngredientCard` objects.